doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+
+.. note::
+    It is possible that a single visit may have an under-sampled PSF, in which case one can expect artifacts and erratic behavior from the *scarlet* models.
+    This problem is not isolated to the deblender, as under-sampled PSFs cause problems in many other places further upstream and will cause much larger issues than the deblender results.
+    However, there are currently no plans to use *scarlet* on single visits and it is expected that co-adds will always be well-sampled.
+
+.. _model_init:
+
+Initialization
+^^^^^^^^^^^^^^
+
+One way to think of the initial *scarlet* models is that they are basically the same as the templates generated by the single band deblender (see `single-band_deblending`_) with one key difference: because the *scarlet* models are obtained using an iterative optimization procedure, using the assumption of monotonicity from the peak location is a well-defined and useful (perhaps necessary) constraint.
+
+For *scarlet* we have chosen the *𝛘²* (inverse variance-weighted or Kaiser) co-add to initialize our morphologies.
+While this is not a truly accurate model (since each observation uses a different seeing), it gives us a fast and efficient method for generating a detection image.
+The main downfall of this choice is that our model space has a much narrower seeing than the detection image, meaning our initial models will always be "puffy."
+We have tried numerous deconvolution and approximation methods but so far we have not found any method that provides enough of an improvement to overcome the runtime required to generate the detection image.
+In other words, every method that we have come up with to generate a better detection image is more expensive computationally than just running the scarlet optimizer for a few extra iterations to converge to the same solution.
+
+Once we have fit the morphology for each component of each source, we use a least-squares algorithm to simultaneously fit the SED of each source *in scarlet model space*.
+Because we normalize the morphology, including the intensity information in the SED, this is different than the observed SED, as the PSF in each band will distribute the model flux differently.
+In this case the computational savings of the least-squares SED fit *is* significant enough to justify its use.
+
+.. note::
+    For *very* large (approx *1k* × *1k* pixel) blends the least squares fit runs into memory issues, since the size of the array to fit is roughly ``number of sources × number of bands × width × height)``.
+    In that case setting :attr:`~lsst.meas.extensions.scarlet.ScarletDeblendConfig.setSpectra` to ``False`` is recommended.
+
+.. note::
+    An interesting fact about optimization in scarlet is that the best initialization (in terms of 𝛘²) will not necessarily result in the fastest runtime or even the most optimal fit.
```

Comment:
Isn't this a general result of being trapped in a local minimum?

Reply:
True, but I think this is more than that. The Adaprox algorithm that is being used right now actually prefers solutions further from the true minimum and will jump out of a good solution in the first iteration. This is different than the proximal gradient method (PGM) and other optimization algorithms, which usually step closer in the gradient direction, even in the first step. I've been doing some testing lately and Adaprox seems even more finicky than I had previously thought, and with the PIFF PSFs seems to perform worse than the standard PGM. But that's a discussion for another day...

Reply:
Ok, then perhaps just mention that this is optimizer-dependent and not some quirk of other aspects of Scarlet.